frontend/src/components/pages/AdminDashboard/ViewDonationsAndCheckIns.tsx

```diff
@@ -127,6 +131,11 @@ const ViewDonationsAndCheckIns = ({
     ]);
   };
 
+  const handleCSVDownload = async () => {
+    const csvScheduleData = await getScheduleCSVData();
+    downloadCSV(csvScheduleData, "foodRescues");
+  };

```

Comment:
nit: why don't we keep this function inside of `CSVUtils` and expose a parameter to download the correct checkin/scheduling csv?